### PR TITLE
[tempo-distributed] Update tempo-distributed to have metrics_ingestion_time_range_slack

### PR DESCRIPTION
**Pull request by EStork09**
metrics_ingestion_time_range_slack = 30s is the default and added to the tempo configuration
Addresses #2432

### CODE REVIEW

**Review by zanhsieh — CHANGES_REQUESTED** [review]
@EStork09 
Thanks for the contribution. Can you bump the version in Chart.yaml to latest? So later CI can merge this PR once approved.

**Comment by EStork09** [comment]
@zanhsieh I synced my fork with main, let me know if there is anything else!

**Review by zanhsieh — APPROVED** [review]

**Pull request merged**
